src/vma/sock/sock-redirect.cpp

```diff
@@ -776,6 +819,9 @@ int setsockopt(int __fd, int __level, int __optname,
 		ret = p_socket_object->setsockopt(__level, __optname, __optval, __optlen);
 	}
 	else {
+		BULLSEYE_EXCLUDE_BLOCK_START
```

Comment:
Explanation understood. Let's remain with your original code. No need to change.